spec/models/destination_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Destination, type: :model do
+    let!(:destination_data) { create(:destination) }
+
+    describe 'validations' do
+        context 'city' do
+            it { should validate_presence_of(:city) }
```

Comment:
Please follow the link from Road map with good practice on rspec, you will find that the new syntax is expected or is_expected. Take your time and read about this because they are important.